packaging/nfpm/mimir/mimir.service

```diff
@@ -11,6 +11,8 @@ After=network-online.target
 
 [Service]
 Restart=always
+Type=notify
+TimeoutStartSec=3min
```

Comment:
good job. for maintainers here is the doc:
* https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#Type=
* https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#TimeoutStartSec=

Reply:
3 minutes looks a bit tight to me. Some components will spend a minute just waiting for others to come back up. New store-gateways for example need to then download index headers, which can easily exceed 10 minutes. Can we set this to 20min maybe?

Reply:
or if the setting is not required maybe we can unset it? We deploy Mimir without a liveness probe in k8s and it generally must always either start or fail, but never get stuck.

Reply:
I'm in favor of `20min` by default.
as pointed by @pbailhache the command `systemctl start mimir` will hang for `TimeoutStartSec` value.